authors/apps/authentication/tests/test_views.py

```diff
@@ -219,6 +219,26 @@ def test_hacking_user_can_crash_login(self):
                          u'This field is required.'])
 
 
+class TestUserRetrieve(BaseTest):
```

Comment:
Are these changes part of the PR

Reply:
As for the 404 error comment by @lytes20, I have failed to respond back directly to his comment since it provides no text box input. But, for @lytes20's comment, Authors Haven currently has no API endpoint for the index route and no 404 error page catching response so far.